### DISCUSSION TITLE
brew dockutil not working on macOS 12.3 - need to upgrade Homebrew version from 2 to 3

### DISCUSSION THREAD

**Discussion by karnaker** [Homebrew/discussions#3106] [Tap maintenance and Homebrew development]
Could you please update the [dockutil homebrew installation](https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/dockutil.rb) so that it installs version 3 instead of 2?

Version 2 doesn't even work on standard macOS Monterey 12.3 anymore, since [python 2 has been removed](https://www.macrumors.com/2022/01/28/apple-removing-python-2-in-macos-12-3/#:~:text=Apple%20will%20no%20longer%20bundle,security%20patches%2C%20or%20other%20changes.).

<img width="740" alt="Screen Shot 2022-03-23 at 4 09 03 PM" src="https://user-images.githubusercontent.com/35387991/159788910-d47988e0-b35a-41be-aee8-533a369f0a73.png">

Thanks for making it easy to install this great tool! ❤️

PS I think this discussion may be of interest to the following (but could be wrong):
@Amar1729
@Mellbourn (I lifted this discussion from your [issue](https://github.com/kcrawford/dockutil/issues/127))
@samford

**Comment by SMillerDev** [comment]
A pull request is already in progress. But generally https://docs.brew.sh/Updating-Software-in-Homebrew applies

**Reply by karnaker** [reply]
Thanks @SMillerDev for the link to the homebrew doc--that's helpful to know! I just tried searching for the pull request, but could not find it. If possible, would you kindly let me know the steps to find the pull request, or send me a link to the pull request, whichever is easier for you?

**Reply by carlocab** [reply]
You can try https://github.com/Homebrew/homebrew-core/pulls?q=is%3Apr+is%3Aopen+dockutil

**Reply by karnaker** [reply]
Thank you @carlocab !
